infra/auth0/appkom.tf

```diff
@@ -1,5 +1,6 @@
 resource "auth0_client" "appkom_opptak" {
   allowed_clients = []
+    cross_origin_auth = true # this is set to avoid breaking client. It was set in auth0 dashboard. Unknown motivation.
```

Comment:
Er ikke dette bare relevant for innloggings-forms som er på f.eks. vinstraff.no eller online.ntnu.no (embedded/custom, hvor du ikke går til et annet domene for å logge inn)? Vi bruker jo alltid universal login hvor du går til auth.onn, og da tror jeg ikke denne er relevant?

https://auth0.com/docs/authenticate/login/cross-origin-authentication